subprojects/platform-native/src/main/java/org/gradle/nativeplatform/platform/internal/ArchitectureInternal.java

```diff
@@ -31,5 +31,8 @@ enum InstructionSet { X86, ITANIUM, PPC, SPARC, ARM }
     boolean isIa64();
 
     @Internal
-    boolean isArm();
```

Comment:
Could you keep this (for backwards compatibility)?  We'd have three methods then:

isArm - true when on ARM (32 or 64)
isArm32 - true when on ARM 32-bit only
isArm64 - true when on ARM 64-bit only

Reply:
Done 👍

Reply:
Sorry for a bit stupid question, but what's the purpose of preserving backward compatibility in class with 'Internal' in name in a package with 'internal' in name?
Is there any data that it's used outside of this repo?

Reply:
Groovy can access these properties, and I think at one point some parts of the c++ documentation used them. So it’s better to not explicitly break anyone using it, especially when keeping it working is fairly easy.